helix-term/src/keymap.rs

```diff
@@ -682,6 +682,10 @@ impl Default for Keymaps {
                 "C-j" | "j" | "down" => jump_view_down,
                 "C-k" | "k" | "up" => jump_view_up,
                 "C-l" | "l" | "right" => jump_view_right,
+                "n" => { "New split scratch buffer"
```

Comment:
I know doom emacs for this `space w n` will split vertical, not sure if we want so many layers here.

Reply:
I am probably biased as I rarely use the scratchpad rarely but I think this would be fine without default keybindings at all. I think the typable commands are more appropriate, especially given how many keypresses these keybindings take.

Reply:
It's definitely a tad deep! My argument for keeping it in here would be that when I first installed helix and played around with it, I was very much inclined to pressing `space` and discovering commands through the dialog. I naturally began to start using `space w` to find window commands, and felt like this was an appropriate place to put them since the other split commands were here too.

Reply:
Hmm good point, the discoverability is nice with the `space` popups. Although it also possible to discover things through the command picker (#1400), but I don't remember if that shows typable commands. I don't have strong feelings about the bindings, I would feel ok with these 👍